packages/unravel/app/blog/blog-data.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from "zod";
+import { AtpAgent } from "@atproto/api";
+const agent = new AtpAgent({
+  service: "https://hydnum.us-west.host.bsky.network",
+});
+
+// Schemas
+const Blog = z.object({
+  value: z.object({
+    content: z.string(),
+    title: z.string(),
+    createdAt: z.coerce.date(),
+  }),
+  cid: z.string(),
+});
+
+const BlogArray = z.array(Blog);
+
+const repo = "did:plc:klmr76mpewpv7rtm3xgpzd7x";
+const collection = "com.whtwnd.blog.entry";
+
+// Functions
+export async function listRecords() {
```

Comment:
Dunno if it matters but maybe ‘getRecords’ or ‘getBlogs’ is a better name here

Reply:
`listBlogs` sounds good.